WordPress/gutenberg
# Icons: Add align-item related icons

## What?

This PR adds the following five icons:

- ~~`alignItemBottom`~~ `justifyBottom`
- ~~`alignItemCenter`~~ `justifyCenterVertical`
- ~~`alignItemSpaceBetween`~~ `justifySpaceBetweenVertical`
- ~~`alignItemStretch`~~ `justifyStretchVertical`
- ~~`alignItemTop`~~ `justifyTop`


## Why?

Since we already have icons related to justify, I thought it would make sense to add icons related to align as well:

![image](https://github.com/user-attachments/assets/b2ca7d60-b0ec-4294-acb1-073a9352cd1f)


## How?

There are already icons with the `align` prefix:

![image](https://github.com/user-attachments/assets/a25a794d-19ca-4452-a6e1-e60ed9b32657)

The four icons other than `alignNone` should ideally have the `textAlign` prefix instead of the align `prefix`, but this cannot be changed to ensure backward compatibility.

~~Therefore, I used the `alignItem` prefix for the newly added icons.~~ I added the `justify` prefix to the newly added icon to group it with other justify-related icons. However, because there are name conflicts with the three icons `center`, `spaceBetween`, and `stretch`, I added the suffix `vertical`.

## Testing Instructions

- Storybook:
  - Run `npm run storybook:dev`
  - Access http://localhost:6005/?path=/story/icons-icon--library.
- Block Editor:
  - Confirm that blocks using this icon work as before.
  - Insert a Row block and check the control on the block toolbar.

## Screenshots or screencast 

![image](https://github.com/user-attachments/assets/2451baf2-46af-4df2-b480-73c6e6f2f194)
